course5/homework05/vector2d.h

```diff
@@ -0,0 +1,68 @@
+#ifndef vector2d_h
```

Comment:
```#pragma once``` is easier, faster and safer header guard. It is not standard C++, but is implemented by all major compilers